root/etc/e-smith/templates/etc/postfix/main.cf/40enableTLS

```diff
@@ -0,0 +1,27 @@
+
+#
+# TLS
+#
+
+# SMTPD
+smtpd_tls_security_level = may
+smtpd_tls_auth_only = no
+smtpd_tls_key_file = /etc/postfix/postfix.key
+smtpd_tls_cert_file = /etc/postfix/postfix.crt
+smtpd_tls_received_header = yes
+smtpd_tls_loglevel = 1
+smtpd_tls_session_cache_timeout = 3600s
+tls_random_source = dev:/dev/urandom
+
+# SMTP
+smtp_tls_security_level = may
+smtp_tls_loglevel = 1
+smtp_tls_cert_file = /etc/postfix/postfix.crt
+smtp_tls_key_file = /etc/postfix/postfix.key
+
+# Force TLSv1
+smtpd_tls_mandatory_protocols=!SSLv2,!SSLv3
```

Comment:
Actual value should be controlled by a centralized prop. We should define that prop, for instance `tls/policy` - allowed values:

* `legacy` the current default settings, before Mar 14 2018
* `20180314`, the new policy we're implementing
* future policies identified by release date

Each package is responsible to implement the policy in its templates, by providing the actual settings for OpenSSL (or any other TLS library).